setup.py

```diff
@@ -25,8 +25,8 @@
       packages=['s3fs'],
       python_requires='>= 3.6',
       install_requires=[open('requirements.txt').read().strip().split('\n')],
-      extras_require = {
-          'awscli': ['aiobotocore[awscli]'],
-          'boto3': ['aiobotocore[boto3]'],
+      extras_require={
+          'awscli': ['aiobotocore[awscli] ~= 1.4.0'],
```

Comment:
Can this be generated from the version in requirements.txt?

Reply:
Sure, I can give that a shot.